assets/src/dashboard/types.js

```diff
@@ -25,7 +25,7 @@ import StoryPropTypes from '../edit-story/types';
 
 export const StoryPropType = PropTypes.shape({
   id: PropTypes.number.isRequired,
-  status: PropTypes.string.isRequired,
+  status: PropTypes.string,
```

Comment:
want to do `PropTypes.oneOf(['active', 'draft', 'template'])` ?

Reply:
Hmmm... I like that!  I'll make the change!